[2022-08-17 19:53:42,795] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Deepface.validate manual__2022-08-17T19:53:39.247876+00:00 [queued]>
[2022-08-17 19:53:43,038] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Deepface.validate manual__2022-08-17T19:53:39.247876+00:00 [queued]>
[2022-08-17 19:53:43,038] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 19:53:43,038] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-08-17 19:53:43,038] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-17 19:53:43,070] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): validate> on 2022-08-17 19:53:39.247876+00:00
[2022-08-17 19:53:43,082] {standard_task_runner.py:52} INFO - Started process 3240 to run task
[2022-08-17 19:53:43,086] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Deepface', 'validate', 'manual__2022-08-17T19:53:39.247876+00:00', '--job-id', '59', '--raw', '--subdir', 'DAGS_FOLDER/deepface.py', '--cfg-path', '/tmp/tmphffgk6ma', '--error-file', '/tmp/tmptbxkkaex']
[2022-08-17 19:53:43,088] {standard_task_runner.py:80} INFO - Job 59: Subtask validate
[2022-08-17 19:53:43,189] {task_command.py:370} INFO - Running <TaskInstance: Deepface.validate manual__2022-08-17T19:53:39.247876+00:00 [running]> on host d254c144b912
[2022-08-17 19:53:43,374] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Deepface
AIRFLOW_CTX_TASK_ID=validate
AIRFLOW_CTX_EXECUTION_DATE=2022-08-17T19:53:39.247876+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-17T19:53:39.247876+00:00
[2022-08-17 19:53:51,513] {logging_mixin.py:115} INFO - 1/1 [==============================] - ETA: 0s
[2022-08-17 19:53:51,514] {logging_mixin.py:115} INFO - 1/1 [==============================] - 1s 775ms/step
[2022-08-17 19:53:51,847] {logging_mixin.py:115} INFO - 1/1 [==============================] - ETA: 0s
[2022-08-17 19:53:51,847] {logging_mixin.py:115} INFO - 1/1 [==============================] - 0s 246ms/step
[2022-08-17 19:53:54,778] {logging_mixin.py:115} INFO - Email sent successfully!
[2022-08-17 19:53:54,779] {python.py:173} INFO - Done. Returned value was: None
[2022-08-17 19:53:54,859] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Deepface, task_id=validate, execution_date=20220817T195339, start_date=20220817T195342, end_date=20220817T195354
[2022-08-17 19:53:55,052] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-17 19:53:55,089] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
